libc/src/stdlib/qsort_data.h

```diff
@@ -92,6 +92,12 @@ class Array {
   Array make_array(size_t i, size_t s) const {
     return Array(get(i), s, elem_size, compare);
   }
+
+  // Reset this Array to point at a different interval of the same items.
+  void reset_bounds(uint8_t *a, size_t s) {
```

Comment:
This method, and the others should probably have the `LIBC_INLINE` macro added.
https://libc.llvm.org/dev/code_style.html#inline-functions-and-variables-defined-in-header-files